samples/client/echo_api/python-nextgen/openapi_client/api/query_api.py

```diff
@@ -18,6 +18,7 @@
 from pydantic import validate_arguments, ValidationError
 from typing_extensions import Annotated
 
+
```

Comment:
I wonder if it's possible to remove this empty line. Linters may complain about it.